src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -352,6 +352,20 @@ private static bool TryGetTextForKeyword(SyntaxToken token, ISyntaxFactsService
                 }
             }
 
+            if (token.IsKind(SyntaxKind.DefaultKeyword)) {
+                if (token.Parent is DefaultSwitchLabelSyntax)
+                {
+                    text = Keyword("defaultcase");
+                    return true;
+                }
+
+                if (token.Parent is DefaultExpressionSyntax || token.Parent is LiteralExpressionSyntax)
```

Comment:
```suggestion
                if (token.Parent is DefaultExpressionSyntax or LiteralExpressionSyntax)
```

Reply:
I believe `or` is only valid in VB, isn't it?

Reply:
It's a new C# 9 feature. See the **Pattern Combinators** section in https://github.com/dotnet/csharplang/issues/2850

Reply:
Ahh, I need to get up to speed on C# 9.  Thanks for pointing that out.